llvm/utils/UpdateTestChecks/common.py

```diff
@@ -773,12 +781,7 @@ def process_run_line(
             )
             if "analysis" in m.groupdict():
                 analysis = m.group("analysis")
-                supported_analyses = {
-                    "cost model analysis",
-                    "scalar evolution analysis",
-                    "loop access analysis",
-                }
-                if analysis.lower() not in supported_analyses:
```

Comment:
I don't know why we need to normalize to lower case, but the change seems fine.

Reply:
I don't think this normalization added anything. It did make it harder to search for the list of approved analyses though, so I used the original spelling from the code now.